### PR TITLE
refactor: use Defilama chainId to name map

### PR DESCRIPTION
**Pull request by Chef-Yogi**
### Summary
🆕🔄📝

This pull request refactors the codebase to use a common object for mapping chain IDs to chain names. It creates a new file `chainIdToName.ts` that exports this object and updates the `farms` package to import and use it. This improves code readability and consistency.

> _Sing, O Muse, of the cunning coder who devised_
> _A constant object, `chainIdTOName`, to map the chains_
> _Of various networks, where gods and mortals trade and strive,_
> _And where the blockchain's fate and fortune ever reigns._

### Walkthrough
*  Add a new constant object `chainIdTOName` that maps chain IDs to chain names for various blockchain networks ([link](https://github.com/pancakeswap/pancake-frontend/pull/7580/files?diff=unified&w=0#diff-200204a6957289727866e8ad5353d96066f859d6024dd0e53c69569ba05b30faR1-R94))
*  Refactor the existing constant `CHAIN_ID_TO_CHAIN_NAME` to use the `chainIdTOName` object instead of hardcoding the chain names ([link](https://github.com/pancakeswap/pancake-frontend/pull/7580/files?diff=unified&w=0#diff-a83c9abf17a3d9a58cea158de8ef950f87ff748b95d6e3119c3f46d4239ddd06L23-R32))
*  Import the `chainIdTOName` object into the `index.ts` file where it is used by other constants and types ([link](https://github.com/pancakeswap/pancake-frontend/pull/7580/files?diff=unified&w=0#diff-a83c9abf17a3d9a58cea158de8ef950f87ff748b95d6e3119c3f46d4239ddd06R14))

### CODE REVIEW

File: packages/farms/constants/common/chainIdToName.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+export const chainIdTOName = {
```
Comment (ChefMomota) [inline packages/farms/constants/common/chainIdToName.ts:1]: Add the link annotation here,  where can u get this

File: packages/farms/constants/common/index.ts [hunk 2]
```diff
@@ -20,15 +21,15 @@ export type PriceHelper = {
...
+  [ChainId.BSC]: chainIdTOName[ChainId.BSC],
+  [ChainId.ETHEREUM]: chainIdTOName[ChainId.ETHEREUM],
+  [ChainId.GOERLI]: chainIdTOName[ChainId.ETHEREUM],
+  [ChainId.BSC_TESTNET]: chainIdTOName[ChainId.BSC],
+  [ChainId.POLYGON_ZKEVM]: chainIdTOName[ChainId.POLYGON_ZKEVM],
+  [ChainId.ZKSYNC]: chainIdTOName[ChainId.ZKSYNC],
   [ChainId.POLYGON_ZKEVM_TESTNET]: '',
   [ChainId.ZKSYNC_TESTNET]: '',
-  [ChainId.ARBITRUM_ONE]: 'arbitrum',
+  [ChainId.ARBITRUM_ONE]: chainIdTOName[ChainId.ARBITRUM_ONE],
```
Comment (ChefMomota) [inline packages/farms/constants/common/index.ts:32]: Can we add the arb testnet?

File: packages/farms/constants/common/index.ts [hunk 3]
```diff
@@ -20,15 +21,15 @@ export type PriceHelper = {
...
-  [ChainId.POLYGON_ZKEVM]: 'polygon_zkevm',
-  [ChainId.ZKSYNC]: 'era',
+  [ChainId.BSC]: chainIdTOName[ChainId.BSC],
+  [ChainId.ETHEREUM]: chainIdTOName[ChainId.ETHEREUM],
+  [ChainId.GOERLI]: chainIdTOName[ChainId.ETHEREUM],
+  [ChainId.BSC_TESTNET]: chainIdTOName[ChainId.BSC],
+  [ChainId.POLYGON_ZKEVM]: chainIdTOName[ChainId.POLYGON_ZKEVM],
+  [ChainId.ZKSYNC]: chainIdTOName[ChainId.ZKSYNC],
   [ChainId.POLYGON_ZKEVM_TESTNET]: '',
   [ChainId.ZKSYNC_TESTNET]: '',
```
Comment (ChefMomota) [inline packages/farms/constants/common/index.ts:31]: fill it

**Comment by ChefMomota** [comment]
Also update the `fetchTokenUSDValue` from `packages/utils/llamaPrice.ts`

**Review by chefjackson — CHANGES_REQUESTED** [review]
We have `@pancakeswap/chains` now. Let's move the config there

**Comment by chefjackson** [comment]
Close this one as we do the refactor here https://github.com/pancakeswap/pancake-frontend/pull/8152

**Pull request closed without merging**